### ISSUE TITLE
docker daemon often fails to start, quick fix

### ISSUE THREAD

**Issue by IanMcDowell** [DataBiosphere/toil#2303]
This is a common scenario for me:
```
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
INFO:toil.provisioners.node:... Still waiting for docker daemon, trying in 10 sec...
```
And it can be fixed easily by giving docker a little nudge, e.g. by replacing:
```
                output = self.sshInstance('/usr/bin/ps', 'aux', sshOptions=['-oBatchMode=yes'], user=keyName)
                if 'dockerd' in output:
```
with:
```
                startup = self.sshInstance('/usr/bin/docker', 'ps', sshOptions=['-oBatchMode=yes'], user=keyName)
                output = self.sshInstance('/usr/bin/systemctl', 'show', '--property', 'ActiveState', 'docker', sshOptions=['-oBatchMode=yes'], user=keyName)
                if 'ActiveState=active' in output:
```
in `provisioners/node.py`

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/TOIL-317)
┆Issue Number: TOIL-317

**Comment by joelarmstrong** [comment]
Thanks! It sounds stupid (and it is!), but another quick fix is to resize your terminal window (see #2250 for the fix we chose). This was a regression in one of the newer toil versions. Hopefully it'll be fixed with the next release.

**Comment by DailyDreaming** [comment]
@IanMcDowell Toil 3.17.0 was released. Let us know if you run into any further trouble.

Closing this issue.
